packages/core/src/components/RadioButton/RadioButton.tsx

```diff
@@ -114,6 +127,7 @@ const RadioButton: VibeComponent<RadioButtonProps, HTMLElement> & object = forwa
               type="radio"
               value={value}
               name={name}
+              autoFocus={autoFocus}
```

Comment:
I'm not sure if both methods are needed, the `useEffect` and the `autoFocus`.
I know that `TextField` implements it using the `useEffect` with the animation frame, which I guess is for a reason.
@talkor wdyt?

Reply:
Both should work but IMO it's better to just use the `autoFocus` prop which is an HTML attribute which makes the useEffect redundant in this case

Reply:
@chensara let's do the following
use the native `autoFocus` instead of the `useEffect`.

- let's release a prerelease
- or just use `yarn pack` on `monday-ui-react-core`'s `packages/core` and use it in your local version
- you can also use `yarn link`, which is better IMO, with your MF. 
Make sure that you run the command using the same node version that is used on your MF, in Vibe we're using node v18.12.1
Also make sure to run `yarn link monday-ui-react-core` afterwards on your MF of course
Notice in this case that you need to run `yarn build` for every change to take effect on your MF

if it works for you with the native approach, amazing. if not, try the same with the `useEffect`

Reply:
So I first implemented using only `autoFocus` attr but it doesn't work in stroybook so I added the useEffect and it worked. Should I delete `autoFocus` attr?

Reply:
when I've checked with the `autoFocus` it seems like working